### PR TITLE
Update submodule `schemas`

### PR DESCRIPTION
**Pull request by KellerFuchs**

### CODE REVIEW

**Comment by ghost** [comment]
@KellerFuchs Did you want to do something more here before the merge?

**Comment by KellerFuchs** [comment]
@dpflug Nope, this is just stuck waiting for someone to approve the PR...

**Review by RyanSquared — APPROVED** [review]

**Pull request merged**
